GOVERNANCE.md

```diff
@@ -0,0 +1,131 @@
+
+### Technical decisions
+Technical decisions that only affect a single project are made informally by the maintainer of this project, and [lazy consensus](#lazy-consensus) is assumed. Technical decisions that span multiple parts of the Fn Project should be discussed and made in the [Fn Project forums](https://forums.fnproject.io/).
+
+Decisions are usually made by [lazy consensus](#lazy-consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### Governance changes
+Material changes to this document are discussed publicly in the [Fn Project forums](https://forums.fnproject.io/). Any change requires a [supermajority](#supermajority-vote) in favor. Editorial changes may be made by [lazy consensus](#lazy-consensus) unless challenged.
+
+### Other matters
+Any matter that needs a decision, including but not limited to financial matters, may be called to a vote by any member if they deem it necessary. For financial, private, or personnel matters, discussion and voting takes place in the [private Fn Team category in the Fn Project forums](https://forums.fnproject.io/c/fn-team) otherwise in the public [Fn Project forums](https://forums.fnproject.io/).
+
+## Voting
+The Fn Project usually runs by informal consensus, however sometimes a formal decision must be made.
+
+Depending on the subject matter, as laid out above, different methods of voting are used.
+
+For all votes, voting must be open for at least one week. The end date should be clearly stated in the call to vote. A vote may be called and closed early if enough votes have come in one way so that further votes cannot change the final decision.
+
+In all cases, all and only [team members](#team-members) are eligible to vote, with the sole exception of the forced removal of a team member, in which said member is not eligible to vote.
+
+Discussion and votes on personnel matters (including but not limited to team membership and maintainership) are held in private in the [private Fn Team category in the Fn Project forums](https://forums.fnproject.io/c/fn-team). All other discussion and votes are held in public in the [Fn Project forums](https://forums.fnproject.io/).
+
+For public discussions, anyone interested is encouraged to participate. Formal power to object or vote is limited to [team members](#team-members).
+
+### Lazy consensus
+The default decision making mechanism for the Fn Project is [lazy consensus](https://couchdb.apache.org/bylaws.html#lazy). This means that any decision on technical issues is considered supported by the [team](https://forums.fnproject.io/groups/FnTeam) as long as nobody objects.
+
+Silence on any consensus decision is implicit agreement and equivalent to explicit agreement. Explicit agreement may be stated at will. Decisions may, but do not need to be called out and put up for decision in the [Fn Project forums](https://forums.fnproject.io/) at any time and by anyone.
```

Comment:
I don't follow the meaning of "put of for decision in the forums" here.

Reply:
s/decision/discussion/